editor/src/components/navigator/navigator-item/expandable-indicator.tsx

```diff
@@ -12,13 +12,14 @@ interface ExpandableIndicatorProps {
   onMouseDown?: (e: any) => void
   onClick?: (e: any) => void
   testId?: string
+  style?: React.CSSProperties
 }
 
 export const ExpandableIndicator: React.FunctionComponent<ExpandableIndicatorProps> = betterReactMemo(
   'ExpandableIndicator',
   (props) => {
     return (
-      <div data-testid={props.testId} style={{ width: 16, height: 16 }}>
+      <div data-testid={props.testId} style={{ width: 16, height: 16, ...props.style }}>
```

Comment:
I don't think this will make any(or very little) difference to this because `ExpandableIndicator` is memoised, but this is a pattern that is better to avoid because it results in the value of `style` changing each time.

Reply:
Good shout. It doesn't really require separate styling anyway, we can have a SubduedExpandableIndicator component. Will fix next week.